### PR TITLE
DBZ-6321:  Document Requirements for multiple…

### PR DESCRIPTION
**Pull request by ironakj**
chore(Postgres-Document): Document Requirements for multiple connectors on same db host

Addresses/Closes DBZ-6321

### CODE REVIEW

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc [hunk 1]
```diff
@@ -2262,6 +2262,19 @@ Again, regularly emitting events solves the problem.
...
 endif::community[]
 
+
+// Type: concept
+// ModuleID: postgresql-deploying-multiple-connectors
+// Title: Setting up multiple connectors for same database server
+[[postgresql-deploying-multiple-connectors]]
+=== Setting up multiple connectors for same database server
+
+It is highly possible that a single Postgres cluster hosts multiple databases (e.g. if the cluster is shared among multiple services, each service has its own database), since a replication slot only streams data from a single database, you'll need a replication slot for each database, or rather for each connector. As even if two connectors are pointing to the same database, they may have different table filters and other configs. To avoid them interfering with each other and thus causing unexpected behavior, it's best to keep unique slot for each connector - the general idea being to keep one replication slot per consumer.
```
Comment (Naros) [inline documentation/modules/ROOT/pages/connectors/postgresql.adoc:2272]: I would reword this because each connector requires its own unique replication slot regardless of the number of databases hosted by the PG cluster.  Keep in mind, a slot is effectively a pointer into the WAL and each connector can be consuming from different positions, and needs to independently manage that pointer.  Furthermore, PG doesn't even allow you to use a replication slot if its already active, i.e. in use by another process.

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc [hunk 2]
```diff
@@ -2262,6 +2262,18 @@ Again, regularly emitting events solves the problem.
...
+
+Since a replication slot can only be used by a single connector, it is essential to create a unique replication slot for each Debezium connector. Although when a connector is not active, Postgres may allow other connector to consume the replication slot - which could be dangerous as it may lead to data loss as a slot will emit each change just once [link:https://www.postgresql.org/docs/10/logicaldecoding-explanation.html#LOGICALDECODING-REPLICATION-SLOTS[See More]].
+
+In addition to replication slot, {prodname} uses publication to stream events when using the `pgoutput` plugin. Similar to replication slot, publication is at database level and is defined for a set of table. Thus, you'll need a unique publication for each connector, unless the connectors work on same set of tables. For more information about the options for enabling {prodname} to create publications, see xref:{link-postgresql-connector}#postgresql-publication-autocreate-mode[`publication.autocreate.mode`]
```
Comment (Naros) [inline documentation/modules/ROOT/pages/connectors/postgresql.adoc:2273]:
```suggestion
In addition to replication slot, {prodname} uses publication to stream events when using the `pgoutput` plugin. Similar to replication slot, publication is at database level and is defined for a set of tables. Thus, you'll need a unique publication for each connector, unless the connectors work on same set of tables. For more information about the options for enabling {prodname} to create publications, see xref:{link-postgresql-connector}#postgresql-publication-autocreate-mode[`publication.autocreate.mode`]
```

**Review by Naros — CHANGES_REQUESTED** [review]
I found one remaining typo; otherwise, this LGTM thus far.

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc (hunk 2 above)
Comment (ironakj) [reply documentation/modules/ROOT/pages/connectors/postgresql.adoc]: Changed

**Review by Naros — APPROVED** [review]
